Remove conan-center default remote (#2212)
conan-io/docs: uploading_packages/remotes.rst

conan-center [deprecated]
-------------------------

> **conan-center** was the official repository but is is **no longer a default remote** in the Conan client and **its usage is completely

Author: Left this section for reference purposes (as some people might still be using the remote until we completely shut it down), but no objections if we want to remove it, as this is documented in previous versions of the documentation (although maybe not so easy to find it).

> **conan-center** was the official repository but is is **no longer a default remote** in the Conan client and **its usage is completely

Reviewer: ```suggestion
**conan-center** was the official repository but is **no longer a default remote** in the Conan client and **its usage is completely
```

discouraged**. This documentation is kept here only for reference purposes.

.. code-block:: bash

    $ conan-center: https://conan.bintray.com [Verify SSL: True]

> It contains all the packages that were in the `conancenter` remote until July 1st (new packages are no longer uploaded to this remote),

Reviewer: This `conancenter` is misleading, as it is the name of the current remote

```suggestion
It contains all the packages that were uploaded to ConanCenter before July 1st (new packages are no longer uploaded to this remote),
```
